taskcluster/ci/toolchain/android.yml

```diff
@@ -23,3 +22,31 @@ linux64-android-sdk-linux-repack:
         toolchain-alias: android-sdk-linux
     treeherder:
         symbol: TL(android-sdk-linux)
+    worker:
+        docker-image: {in-tree: base}
+
+
+linux64-android-gradle-dependencies:
+    description: "Android Gradle dependencies toolchain task"
+    fetches:
+        toolchain:
+            # Aliases aren't allowed for toolchains depending on toolchains.
+            - linux64-android-sdk-linux-repack
+    run:
+        script: android-gradle-dependencies.sh
+        sparse-profile: null
+        resources:
+            - taskcluster/scripts/toolchain/android-gradle-dependencies.sh
+            - taskcluster/scripts/toolchain/android-gradle-dependencies/**
+            - buildSrc/src/main/java/Dependencies.kt
+            # We don't follow 'buildSrc/src/main/java/AndroidComponents.kt' because we only cache
+            # dependencies outside of the ones hosted on maven.mozilla.org.
```

Comment:
What will be the recommended process if we are in a situation where we need to update the toolchain for other reasons (e.g. some aspect of the build changed)?

Reply:
I'll also note that the script used is implicitly tracked, and that adding new files here will also change the digest, triggering a build.